.travis.yml

```diff
@@ -24,8 +24,8 @@ install:
   - bash .travis/install-minikube.sh
 
 script:
-  - make gen-api-fail-if-dirty --always-make
-  - make build
-  - make test
-  - make test-olm
-  - make test-cli-flow
+  - make gen-api-fail-if-dirty --always-make || exit 1
```

Comment:
What went wrong - doesn't make also return 1 when failing?
Or maybe its because the travis script needs explicit exit to be called.

Reply:
it keeps on running unless we exit

Reply:
Travis will run all the scripts sequentially (it is travis design) even if it fails.
the way to tell it not to can be with `|| exit 1` or `|| travis_terminate 1`